### PR TITLE
Start adding start command

### PR DESCRIPTION
**Pull request by sofstam**
### Added

-  `cg workflow taxprofiler start`

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b taxprofiler_start -a
    ```

### How to test

- [x] `cg workflow taxprofiler start richurchin -d`
Screenshot 2023-09-04 at 16 36 10

- [x] `cg workflow taxprofiler start richurchin`-Make sure that config case is created and analysis is being run sussessfully with tower.
Screenshot 2023-09-05 at 11 41 41
Screenshot 2023-09-05 at 11 39 49
- [x] `cg workflow taxprofiler start richurchin --use-nextflow`
Screenshot 2023-09-05 at 15 28 43

## Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/cli/workflow/taxprofiler/base.py [hunk 1]
```diff
@@ -138,3 +136,56 @@ def run(
...
+        from_start=True,
+        profile=profile,
+        nf_tower_id=nf_tower_id,
+        config=config,
+        params_file=params_file,
+        revision=revision,
+        compute_env=compute_env,
+        use_nextflow=use_nextflow,
+        dry_run=dry_run,
+    )
```
Comment (sofstam) [inline cg/cli/workflow/taxprofiler/base.py:191]: I am thinking if I should also make sure here that if I run the command with nextflow then trailblazer does not update any analysis.

Comment (fevac) [reply cg/cli/workflow/taxprofiler/base.py:191]: that is already part of the run command, so you don't need to repeat that here. We are not updating anything here. Only adding a pending case (which happens within the run command)

**Review by fevac — APPROVED** [review]
🌟 Nice! 🌟 There options are very similar (but not identical) between rnafusion and taxprofiler. I wonder we can do something about it. Maybe we can think about it (but outside this pr)

File: cg/cli/workflow/taxprofiler/base.py [hunk 2]
```diff
@@ -138,3 +136,56 @@ def run(
...
+    profile: str,
+    config: str,
+    params_file: str,
+    revision: str,
+    compute_env: str,
+    use_nextflow: bool,
+    nf_tower_id: Optional[str],
+    dry_run: bool,
+) -> None:
+    """Start full workflow for CASE ID."""
```
Comment (henrikstranneheim) [inline cg/cli/workflow/taxprofiler/base.py:168]:
```suggestion
    """Start full workflow for case id."""
```
No need to shout 😄

**Review by henrikstranneheim — APPROVED** [review]
Nice  👍

File: cg/cli/workflow/taxprofiler/base.py (hunk 2 above)
Comment (sofstam) [reply cg/cli/workflow/taxprofiler/base.py:168]: Haha, fair enough :)

File: cg/cli/workflow/taxprofiler/base.py (hunk 1 above)
Comment (sofstam) [reply cg/cli/workflow/taxprofiler/base.py:191]: Right, thanks!

**Comment by sofstam** [comment]
Deploy on stage:
Screenshot 2023-09-05 at 15 49 46

**Pull request merged**